### PR TITLE
Tx 7

### PR DESCRIPTION
**Pull request by osadolortech**

### CODE REVIEW

File: tweetapi/models.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ class ProfileModel(models.Model):
 
 class TweetModel(models.Model):
     owner = models.ForeignKey(User,on_delete=models.CASCADE, related_name='tweet')
-    content = models.TextField(max_length=160,blank=True)
+    content = models.TextField(max_length=160,blank=True,null=True)
```
Comment (iamr0b0tx) [inline tweetapi/models.py]: why null?

File: tweetapi/registration.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    password_confirmation = serializers.CharField(required=True,write_only=True)
+    class Meta:
+        model= User
+        fields=(
+            "first_name","last_name","username","email","password","password_confirmation"
+        )
+        extra_kwargs= {"first_name":{'required':True}, "last_name":{'required':True}}
+    def validate(self, attrs):
+        if attrs["password"] != attrs["password_confirmation"]:
+            raise serializers.ValidationError({"password": "password filed didnt match"})
```
Comment (iamr0b0tx) [inline tweetapi/registration.py]: spelling

**Review by iamr0b0tx — CHANGES_REQUESTED** [review]

File: tweetapi/models.py (hunk 1 above)
Comment (osadolortech) [reply tweetapi/models.py]: fixed

File: tweetapi/registration.py (hunk 2 above)
Comment (osadolortech) [reply tweetapi/registration.py]: fixed

**Comment by osadolortech** [comment]
The null=True was a mistake, I had a case where I was testing the API and I
forgot to remove it.

**Pull request merged**